packages/remix-eslint-config/rules/typescript.js

```diff
@@ -1,18 +1,23 @@
 module.exports = {
   // TODO: These rules might be nice to enable... we should investigate eventually!
+  "@typescript-eslint/array-type": "off",
   "@typescript-eslint/ban-ts-comment": "off",
   "@typescript-eslint/ban-types": "off",
+  "@typescript-eslint/consistent-indexed-object-style": "off",
+  "@typescript-eslint/consistent-type-definitions": "off",
   "@typescript-eslint/no-empty-function": "off",
   "@typescript-eslint/no-empty-interface": "off",
   "@typescript-eslint/no-explicit-any": "off",
   "@typescript-eslint/no-inferrable-types": "off",
   "@typescript-eslint/no-namespace": "off",
   "@typescript-eslint/no-non-null-assertion": "off",
   "@typescript-eslint/no-var-requires": "off",
+  "@typescript-eslint/prefer-function-type": "off",
   "no-var": "off",
   "prefer-rest-params": "off",
 
   // These rules are nice and we want to configure over the defaults
+  "@typescript-eslint/consistent-type-imports": "error",
```

Comment:
We ended up not putting https://typescript-eslint.io/rules/consistent-type-imports actually in the _`stylistic`_ preset. The TS world hasn't solidified on one preferred style.